flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/state/rocksdb/RocksDBStateUploader.java

```diff
@@ -180,6 +196,31 @@ private HandleAndLocalPath uploadLocalFileToCheckpointFs(
         }
     }
 
+    @VisibleForTesting
+    void applyJitter(Consumer<Long> consumer) {
+        if (uploadJitter.isZero()) {
+            return;
+        }
+
+        long milliseconds = random.nextLong(0, uploadJitter.toMillis() + 1);
+        consumer.accept(milliseconds);
+    }
+
+    static class JitterConsumer implements Consumer<Long> {
+
+        @Override
+        public void accept(Long milliseconds) {
+            try {
+                Thread.sleep(milliseconds);
+            } catch (InterruptedException e) {
+                LOG.error(
+                        "Fail to apply {} milliseconds jitter in RocksDBStateUploader.",
```

Comment:
nit: This reads strangely. I am not sure the phrase failed to apply represents an interrupted sleep. imho I would say:

Jitter sleep for n milliseconds was interrupted in RocksDBStateUploader.